### PR TITLE
Esas/display current workspace info prod 11810

### PR DESCRIPTION
**Pull request by esasova**
Before creating a PR, please check that:
- [ ] Code is clean
- [ ] Tests are still working (cypress tests and `yarn test`)
- [ ] Changes don't cause new errors or warnings in browser console
- [ ] Documentation is up-to-date
- [ ] Breaking changes are clearly identified with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

### CODE REVIEW

File: cypress/e2e/brewery/DiscardAndContinue.cy.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ describe('Discard and continue go to workspaces', () => {
 
     Scenarios.getScenarioViewTab(60).should('be.visible');
 
-    Workspaces.getHomeButton().should('be.visible').click();
+    Workspaces.switchToWorkspaceView();
```
Comment (csm-thu) [inline cypress/e2e/brewery/DiscardAndContinue.cy.js:123]: This makes tests much easier to read, thank you for this generic function :smile:

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Review by nborde-CSM — APPROVED** [review]
LGTM :+1:

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Pull request merged**